src/db/SQLiteHelper.java

```diff
@@ -41,7 +44,7 @@ public SQLiteHelper() {
     public void createDatabase(String databaseName) {
         this.databaseName = databaseName;
         try {
-            connection = DriverManager.getConnection("jdbc:sqlite:");
+            connection = DriverManager.getConnection("jdbc:sqlite:"+databaseName+".db");
```

Comment:
oh...I see.
I was converting one dictionary with your program, but the insert operation just endlessly blocked at the 3rd word.
I thought possibly it is your code won't work with my downloaded version of jdbc driver, so I followed jdbc driver's instruction to modify your code, but only to discovered that it your escape sql string don't work quite well.
In my opinion, memory table write then flush to disk is a better idea than my batch insert statment version. But other modifications such as preparedstatement should be adopted.
If we can shake on this, I will remove my batch insert statement code when I'm free.